simulation_ws/src/deepracer_simulation/urdf/racecar.gazebo

```diff
@@ -63,28 +63,23 @@
 <xacro:unless value="$(optenv DISABLE_GAZEBO_CAMERA false)">
   <gazebo reference="camera_link">
     <sensor type="camera" name="zed_camera_left_sensor">
-      <update_rate>60.0</update_rate>
+      <update_rate>15.0</update_rate>
       <!-- math.atan(320 / 687.8065795898438) * 2 -->
       <camera name="zed_camera_left_camera">
-        <horizontal_fov>1.1931415</horizontal_fov>
+        <horizontal_fov>1.13</horizontal_fov>
         <image>
-          <width>160</width>
-          <height>120</height>
-          <format>B8G8R8</format>
+          <width>640</width>
+          <height>480</height>
+          <format>R8G8B8</format>
         </image>
         <clip>
           <near>0.02</near>
           <far>300</far>
         </clip>
-        <noise>
-          <type>gaussian</type>
-          <mean>0.0</mean>
-          <stddev>0.007</stddev>
-        </noise>
       </camera>
       <plugin name="camera_controller" filename="libgazebo_ros_camera.so">
         <alwaysOn>true</alwaysOn>
-        <updateRate>60.0</updateRate>
+        <updateRate>15.0</updateRate>
```

Comment:
It seems we can only process 10 images in a second (https://github.com/aws-robotics/aws-robomaker-sample-application-deepracer/blob/master/simulation_ws/src/sagemaker_rl_agent/markov/environments/deepracer_env.py#L46) Should this be 10?

Reply:
This is 15 because DeepRacer camera can at most do 15 frames per second. The number of inferences we do currently is 10 per second, but that can potentially be changed to 15. At the moment, it skips a few frames.